deepchecks/core/suite.py

```diff
@@ -224,37 +231,42 @@ def to_widget(
         """
         return SuiteResultWidgetSerializer(self).serialize(output_id=unique_id)
 
-    def to_json(self, with_display: bool = True):
+    def to_json(self, with_display: Optional[bool] = None):
         """Return check result as json.
 
         Parameters
         ----------
         with_display : bool
             controls if to serialize display of checks or not
+            (the parameter is deprecated and does not have any effect since version 0.6.4,
+            it will be removed in future versions)
 
         Returns
         -------
         str
         """
-        # TODO: not sure if the `with_display` parameter is needed
-        # add deprecation warning if it is not needed
+        if with_display is not None:
```

Comment:
it should have an effect, it used to have.
probably haven't noticed when I reviewed your refactor

Reply:
fixed